project_start/express-api/routes/post.js

```diff
@@ -30,6 +30,16 @@ router.get('/search/:query', async (req, res, next) => {
   })
 
 
+  // Middleware for getting the current song to limit redundancy
```

Comment:
Remove comments/log statements before checking into the master branch

Reply:
Great suggestion thank you!